src/js/facility-locator/components/markers/TownHall.jsx

```diff
@@ -0,0 +1,30 @@
+import DivMarker from './DivMarker';
+import React, { Component, PropTypes } from 'react';
+
+class TownHall extends Component {
```

Comment:
This was a mock component in case we wanted to use custom Icons. I will probably delete this if we decide on using Numbered Icons—in which case I'll make a `NumberedIcon` component